docs/core/run-time-config/garbage-collector.md

```diff
@@ -0,0 +1,204 @@
+| **app.config for .NET Framework** | [GCHeapCount](../../framework/configure-apps/file-schema/runtime/gcheapcount-element.md) | *number* | 4.6.2 |
+
+### System.GC.HeapAffinitizeMask/COMPlus_GCHeapAffinitizeMask
+
+- Specifies the exact processors that garbage collector threads should use.
+- If processor affinity is disabled by setting `System.GC.NoAffinitize` to `true`, this setting is ignored.
+- Applies to server garbage collection (GC) only.
+- The decimal value is a bit mask that defines the processors that are available to the process. For example, a decimal value of 1023 is equivalent to 0x3FF in hexadecimal notation and 0011 1111 1111 in binary notation. This specifies that the first 10 processors are to be used. To specify the next 10 processors, that is, processors 10-19, specify a decimal value of 1047552, which is equivalent to 0xFFC00 in hexadecimal and 1111 1111 1100 0000 0000 in binary.
+
+| | Setting name | Values | Version introduced |
+| - | - | - | - |
+| **runtimeconfig.json** | `System.GC.HeapAffinitizeMask` | *decimal value* | .NET Core 3.0 |
+| **Environment variable** | `COMPlus_GCHeapAffinitizeMask` | *decimal value* | .NET Core 3.0 |
+| **app.config for .NET Framework** | [GCHeapAffinitizeMask](../../framework/configure-apps/file-schema/runtime/gcheapaffinitizemask-element.md) | *decimal value* | 4.6.2 |
+
+### System.GC.GCHeapAffinitizeRanges/COMPlus_GCHeapAffinitizeRanges
+
+- Specifies the list of processors to use for garbage collector threads.
+- This setting is similar to `System.GC.HeapAffinitizeMask`, except it allows you to specify more than 64 processors.
+- For Windows operating systems, prefix the processor number or range with the corresponding [CPU group](/windows/win32/procthread/processor-groups), for example, "0:1-10,0:12,1:50-52,1:70".
+- If processor affinity is disabled by setting `System.GC.NoAffinitize` to `true`, this setting is ignored.
+- Applies to server garbage collection (GC) only.
+- For more information, see [Making CPU configuration better for GC on machines with > 64 CPUs](https://devblogs.microsoft.com/dotnet/making-cpu-configuration-better-for-gc-on-machines-with-64-cpus/) on Maoni Stephens' blog.
+
+| | Setting name | Values | Version introduced |
+| - | - | - | - |
+| **runtimeconfig.json** | `System.GC.GCHeapAffinitizeRanges` | Comma-separated list of processor numbers or ranges of processor numbers.<br/>Unix example: "1-10,12,50-52,70"<br/>Windows example: "0:1-10,0:12,1:50-52,1:70" | .NET Core 1.0 |
+| **Environment variable** | `COMPlus_GCHeapAffinitizeRanges` | Comma-separated list of processor numbers or ranges of processor numbers.<br/>Unix example: "1-10,12,50-52,70"<br/>Windows example: "0:1-10,0:12,1:50-52,1:70" | .NET Core 1.0 |
+| **app.config for .NET Framework** | N/A | N/A | N/A |
+
+### COMPlus_GCCpuGroup
+
+- Configures whether the garbage collector uses [CPU groups](/windows/win32/procthread/processor-groups) or not.
```

Comment:
I forgot to mention this when we met, we might want to mention the related config for the threadpool so you can have thread pool threads be on all CPUs as well: https://docs.microsoft.com/en-us/dotnet/framework/configure-apps/file-schema/runtime/thread-useallcpugroups-element. we also have the corresponding env var: set COMPlus_Thread_UseAllCpuGroups=1